daphne_worker/src/storage_proxy.rs

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2024 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+//! This is a worker implementation that serves as a proxy to the storage that workers has access
+//! to, i.e., KV and Durable Objects.
+//!
+//! Comunication with this worker is done via http.
+//!
+//! # KV
+//!
+//! The base of all kv request uris is [`KV_PATH_BASE`].
+//!
+//! ## Getting a key
+//!
+//! Make a `GET` request with uri `{KV_PATH_BASE}/path/to/key`.
```

Comment:
We should consider versioning the API, i.e., add to each request.  Here for example:
```suggestion
//! Make a `GET` request with uri `/storage_proxy/v1/{KV_PATH_BASE}/path/to/key`.
```